packages/cfd/src/Constants/jurisdiction-contents/jurisdiction-vanuatu-contents.ts

```diff
@@ -48,7 +48,7 @@ export const getJurisdictionVanuatuContents = ({
             description: localize('Forex, Stock indices, Commodities and Cryptocurrencies'),
             title_indicators: {
                 type: 'displayText',
-                display_text: localize('90+'),
+                display_text: localize('170+'),
```

Comment:
@shontzu-deriv in comparision with design the color should be changed for this content

`design: `
<img width="280" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/d7e49faf-17d1-442f-8b43-f3b349caf7d7">

`test link:`
<img width="296" alt="image" src="https://github.com/binary-com/deriv-app/assets/82078941/8e440b4c-0d29-473c-9eb9-1a4a133c34a0">